### DISCUSSION TITLE
[5.0] node ace make:model command is missing

### DISCUSSION THREAD

**Discussion by codytooker** [adonisjs/core#1360] [Help] [answered]
The v5 preview docs show that one should be able to run `node ace make:model {ModelName}` however I can't get that to work.  After running `node ace -h` I can see that models, migrations and other commands simply don't exists.  Am I doing something wrong?

![image](https://user-images.githubusercontent.com/19712217/88746617-8fc6ac00-d112-11ea-84fe-4edc430697e4.png)

**Comment by codytooker** [comment, marked as answer]
Just figured it out. After installing Lucid you must restart your server.

**Reply by douglasdnzcarvalho** [reply]
Same bug here, thanks for giving the solution!
